catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -178,6 +179,22 @@ public UpdateStorageResponse update(UpdateStorageRequest updateRequest)
             }
         }
 
+        for (ContentItem contentItem : updatedItems) {
```

Comment:
In general we should separate out bug fixes but since this whole PR is really a bug fix I think it is fine as is.